src/drc/datamodel/models.py

```diff
@@ -356,3 +371,35 @@ def unique_representation(self):
             io_id = request_object_attribute(self.object, 'identificatie', self.object_type)
             self._unique_representation = f"({self.informatieobject.latest_version.unique_representation()}) - {io_id}"
         return self._unique_representation
+
+
+class BestandsDeel(models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=_uuid.uuid4,
+        help_text="Unieke resource identifier (UUID4)"
+    )
+    informatieobject = models.ForeignKey(
+        'EnkelvoudigInformatieObjectCanonical', on_delete=models.CASCADE, related_name='bestandsdelen'
+    )
+    index = models.PositiveIntegerField(
+        help_text=_("Een volgnummer dat de volgorde van de bestandsdelen aangeeft.")
+    )
+    grootte = models.PositiveIntegerField(
```

Comment:
@sergei-maertens Should we call it `bestandsdeelomvang` to be similar to `bestandsomvang` on EIO?

Reply:
agree, or maybe just `omvang`?

Reply:
`groote` is renamed to `omvang`